conf/logs.json

```diff
@@ -136,5 +136,52 @@
     "hints": {
       "records": "Records[*]"
     }
+  },
+  "cloudwatch:cloudtrail": {
```

Comment:
this diff got me thinking --  i believe that currently we short circuit to accept the first schema match. this works well as long as there are not multiple schemas that match the event. if there are, this could get sketchy.